x-pack/plugin/data-streams/src/test/java/org/elasticsearch/xpack/core/action/GetDataStreamsRequestTests.java

```diff
@@ -21,22 +21,16 @@
     protected Request createTestInstance() {
         final String[] searchParameter;
         switch (randomIntBetween(1, 4)) {
-            case 1:
-                searchParameter = generateRandomStringArray(3, 8, false, false);
-                break;
-            case 2:
+            case 1 -> searchParameter = generateRandomStringArray(3, 8, false, false);
```

Comment:
Should we make this a proper switch expression, and return a value to assign to `searchParameter`?